convert2rhel/unit_tests/utils_test.py

```diff
@@ -173,7 +173,7 @@ def test_download_pkg_assertion_error(self):
     @unit_tests.mock(system_info, "releasever", "7Server")
     @unit_tests.mock(system_info, "version", namedtuple("Version", ["major", "minor"])(7, 0))
     @unit_tests.mock(utils, "run_cmd_in_pty", RunSubprocessMocked(ret_code=1))
-    @unit_tests.mock(os, "environ", {"CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK": "1"})
+    @unit_tests.mock(os, "environ", {"CONVERT2RHEL_INCOMPLETE_ROLLBACK": "1"})
     def test_download_pkg_failed_download_overridden(self):
         path = utils.download_pkg("kernel")
 
```

Comment:
Now that there are different messages that can be emitted depending on which environment variable is set, you probably want to also assert that the proper message(s) were logged. You can use the `caplog` fixture to achieve that.

Reply:
Yeah I can definitely get that done, Thank you for mentioning that!